src/containers/EditMarkupPage/EditMarkupPage.js

```diff
@@ -0,0 +1,227 @@
+    }
+
+    try {
+      const { draftId, language } = this.props.match.params;
+      const draft = await fetchDraft(draftId, language);
+      this.setState({ draft, status: 'edit' });
+    } catch (e) {
+      handleError(e);
+      this.setState({ status: 'fetch-error' });
+    }
+  }
+
+  saveChanges = async () => {
+    try {
+      const { draftId, language } = this.props.match.params;
+      this.setState({ status: 'saving' });
+      const content = standardizeContent(this.state.draft.content.content);
+      const draft = await updateDraft({
+        id: parseInt(draftId, 10),
+        content,
+        revision: this.state.draft.revision,
+        language,
+      });
+      this.setState({ status: 'edit', draft });
+    } catch (e) {
+      handleError(e);
+      this.setState({ status: 'save-error' });
+    }
+  };
+
+  handleChange = value => {
+    this.setState(prevState => ({
+      draft: updateContentInDraft(prevState.draft, value),
+    }));
+  };
+
+  render() {
+    const { draftId, language } = this.props.match.params;
+    const { status, draft } = this.state;
+
+    if (status === 'access-error') {
+      return (
+        <ErrorMessage
+          draftId={draftId}
+          language={language}
+          messageId="forbiddenPage.description"
+        />
+      );
+    }
+
+    if (status === 'fetch-error') {
+      return (
+        <ErrorMessage
+          draftId={draftId}
+          language={language}
+          messageId="editMarkup.fetchError"
+        />
+      );
+    }
+
+    return (
+      <Trans>
+        {({ t }) => (
+          <Container>
+            <FormHeader
+              title={t('editMarkup.title')}
+              subTitle={t('editMarkup.subTitle')}>
+              <HelpMessage>
+                <p>{t('editMarkup.helpMessage.paragraph1')}</p>
+                <p>{t('editMarkup.helpMessage.paragraph2')}</p>
+              </HelpMessage>
+            </FormHeader>
+            <Suspense fallback={<Spinner />}>
+              <MonacoEditor
+                key={draft ? draft.id + draft.revision : 'draft'}
+                value={draft ? draft.content.content : ''}
+                onChange={this.handleChange}
+              />
+              {status === 'save-error' && (
+                <StyledErrorMessage
+                  css={css`
+                    text-align: left;
+                    margin: ${spacing.normal};
+                  `}>
+                  {t('editMarkup.saveError')}
+                </StyledErrorMessage>
+              )}
+              <Row
+                justifyContent="space-between"
+                css={css`
+                  margin: ${spacing.normal};
+                `}>
+                <PreviewDraftLightbox
+                  label={t('subNavigation.learningResource')}
+                  typeOfPreview="preview"
+                  getArticle={() => {
+                    const content = standardizeContent(
+                      this.state.draft.content.content,
```

Comment:
Ta bort this.state ✌️ Bør du sjekke om draft er defined her også?

Reply:
Tatt bor this.state. At folk trykker på preview før artikkelen er lastet er en edge case jeg ikke synes vi trenger å dekke (ref sjekke defined).